docs/docs/coverage/language/c.md

```diff
@@ -12,12 +12,14 @@ The following table provides an outline of the features Trivy offers.
 
 | Package manager | File           | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position |
 | --------------- | -------------- | :---------------------: | :--------------: | :----------------------------------: | :------: |
-| Conan           | conan.lock[^1] |            ✓            |     Excluded     |                  ✓                   |    ✓     |
+| Conan           | conan.lock[^1] |            ✓ [^2]       |     Excluded     |                  ✓                   |    ✓     |
```

Comment:
Dependency graph is only supported for version v1. Add note please
```suggestion
| Conan           | conan.lock[^1] |            ✓ [^2]       |     Excluded     |                  ✓[^3]                   |    ✓     |
```

Reply:
fixes applied